presto-main/src/main/java/com/facebook/presto/server/TaskUpdateRequest.java

```diff
@@ -64,20 +69,33 @@ public TaskUpdateRequest(
         this.tableWriteInfo = tableWriteInfo;
     }
 
+    //This constructor is for testing purposes only and will soon be deprecated
+    @ThriftConstructor
+    public TaskUpdateRequest(
+            @JsonProperty("session") SessionRepresentation session,
+            @JsonProperty("extraCredentials") Map<String, String> extraCredentials,
+            @JsonProperty("fragment") Optional<byte[]> fragment)
+    {
+        this(session, extraCredentials, fragment, ImmutableList.of(), OutputBuffers.createInitialEmptyOutputBuffers(ARBITRARY), Optional.empty());
+    }
```

Comment:
Curious why have only partially added support. Can we move this out into a separate PR / commit.

Reply:
Removed it for now.